.github/workflows/test.yml

```diff
@@ -14,6 +14,11 @@ on:
     branches:
       - main
       - "[0-9]+.[0-9]+.x"
+    types:
+    - opened  # default
+    - synchronize  # default
+    - reopened  # default
+    - ready_for_review  # used in PRs created from the release workflow
```

Comment:
Note for the future: the code comment refers to the release workflow, but it's actually `update-plugin-list.yml`. We need to update the comment at some point.